cmd/codebuild/reset/config.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"github.com/Optum/dce/pkg/common"
+	"github.com/Optum/dce/pkg/config"
+	"github.com/Optum/dce/pkg/db"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+type resetConfig struct {
+	// Env Vars
+	AccountID             string   `env:"RESET_ACCOUNT"`
+	AdminRoleName         string   `env:"RESET_ACCOUNT_ADMIN_ROLE_NAME"`
+	PrincipalRoleName     string   `env:"RESET_ACCOUNT_PRINCIPAL_ROLE_NAME"`
+	PrincipalPolicyName   string   `env:"RESET_ACCOUNT_PRINCIPAL_POLICY_NAME"`
+	IsNukeEnabled         bool     `env:"RESET_NUKE_TOGGLE"`
+	NukeTemplateDefault   string   `env:"RESET_NUKE_TEMPLATE_DEFAULT"`
+	NukeTemplateBucket    string   `env:"RESET_NUKE_TEMPLATE_BUCKET"`
+	NukeTemplateKey       string   `env:"RESET_NUKE_TEMPLATE_KEY"`
+	ResetCompleteTopicARN string   `env:"RESET_COMPLETE_TOPIC_ARN"`
+	NukeRegions           []string `env:"RESET_NUKE_REGIONS"`
+
+	Session      *session.Session
+	TokenService common.TokenService
+	S3           common.Storager
+	DB           db.DBer
```

Comment:
I'm not sure I like this methodology.  I would rather get the service from the Services itself.  Services.DB().  Adding this here and adding it `WithDB` seems repetitive.